dateparser/search/search.py

```diff
@@ -153,9 +155,12 @@ def search_parse(self, shortname, text, settings):
             languages = [shortname]
             to_parse = original
 
+        
+
+
```

Comment:
Why did you resolve this? I think you should remove these three new newlines.

Reply:
The new lines are removed now